### PR TITLE
Feat: Add aragonOS, dao and minime deploy utils

### PR DESCRIPTION
**Pull request by 0xGabi**
@sohkai this PR is still a draft. I leave you a few questions that I think are the relevant points we should answer. Thanks

### CODE REVIEW

File: packages/deploy-helpers/bases/deploy-apm.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
+const namehash = require('eth-ens-namehash').hash
```
Comment (0xGabi) [inline packages/deploy-helpers/bases/deploy-apm.js:1]: Do you think it makes sense to have scripts to deploy the bases in this package?

File: packages/deploy-helpers/deploy-minime.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const MiniMeTokenFactory = await artifacts.require('@aragon/minime/build/contracts/MiniMeTokenFactory')
```
Comment (0xGabi) [inline packages/deploy-helpers/deploy-minime.js:1]: This util is somewhat relevant. I wonder if it should be part of `@aragon/minime` 🤔

File: packages/deploy-helpers/deploy-dao.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+const Kernel = artifacts.require('@aragon/os/build/contracts/kernel/Kernel')
```
Comment (0xGabi) [inline packages/deploy-helpers/deploy-dao.js:1]: This logic almost all Aragon apps use it. How do you think it would be the best way to expose it?

**Comment by sohkai** [comment]
No, I don't think these should belong in here 😅.

These deployment scripts should either live in the deployments repo or aragen.

**Pull request closed without merging**